marlowe-dashboard-client/src/Types.purs

```diff
@@ -0,0 +1,66 @@
+module Types
+  ( WebData
+  , ContractInstanceId(..)
+  , contractInstanceIdFromString
+  , MarloweParams
+  , ValidatorHash
+  ) where
+
+-- this module is for miscellaneous types that don't belong anywhere else
+-- (or would lead to cyclic dependencies if put somewhere they might more naturall belong)
+import Prelude
+import Data.Generic.Rep (class Generic)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (class Newtype)
+import Data.UUID (UUID, parseUUID)
+import Foreign.Class (class Encode, class Decode)
+import Foreign.Generic (defaultOptions, genericDecode, genericEncode)
+import Marlowe.Semantics (CurrencySymbol)
+import Network.RemoteData (RemoteData)
+import Servant.PureScript.Ajax (AjaxError)
+
+type WebData
```

Comment:
You defined this twice... here and in MainFrame.Types

Reply:
oops :grin: